lib/Shared.js

```diff
@@ -0,0 +1,197 @@
+var parseRange = require("range-parser");
+var pathIsAbsolute = require("path-is-absolute");
+var MemoryFileSystem = require("memory-fs");
+
+module.exports = function shared(context) {
+	var shared = {
```

Comment:
Would you like me to underbar the variable as `_shared` or un-name the function? Or some other resolution?